### PR TITLE
Adds 1.21.3 test support

### PR DESCRIPTION
**Pull request by EquipableMC**
### Description
This simply adds support for 1.21.3 in tests, and also fixes a known issue with EffHealth.sk test where you cannot resurrect a dead mob

---
**Target Minecraft Versions:** 1.21.1
**Requirements:** none
**Related Issues:** none

### CODE REVIEW

File: src/test/skript/tests/syntaxes/effects/EffHealth.sk [hunk 1]
```diff
@@ -10,14 +10,14 @@ test "health effect":
 	spawn cow at location(0, 64, 0, world "world")
 	set {_m} to last spawned cow
 	assert health of {_m} is 5 with "default cow health failed"
-	damage {_m} by 0.5
-	assert health of {_m} is 4.5 with "damage cow failed"
-	damage {_m} by 99
-	assert health of {_m} is 0 with "damage cow failed"
+	damage {_m} by 1
```
Comment (Efnilite) [inline src/test/skript/tests/syntaxes/effects/EffHealth.sk:13]: Should be 0.5 as in the line below you also damage by an int

**Review by Efnilite — CHANGES_REQUESTED** [review]

File: src/test/skript/tests/syntaxes/effects/EffHealth.sk (hunk 1 above)
Comment (EquipableMC) [reply src/test/skript/tests/syntaxes/effects/EffHealth.sk:13]: oh yeah, oops

**Review by Efnilite — APPROVED** [review]

**Review by TheLimeGlass — CHANGES_REQUESTED** [review]
You forgot to update the gradle.properties
```diff
-testEnv=java21/paper-1.21.0
+testEnv=java21/paper-1.21.1
```

**Comment by EquipableMC** [comment]
> You forgot to update the gradle.properties
> 
> ```diff
> 
> -testEnv=java21/paper-1.21.0
> 
> +testEnv=java21/paper-1.21.1
> 
> ```

Ah oops, will do when I can!

**Review by APickledWalrus — CHANGES_REQUESTED** [review]
This should just replace the existing 1.21.0 environment

**Review by TheLimeGlass — APPROVED** [review]

**Review by sovdeeth — APPROVED** [review]

**Comment by EquipableMC** [comment]
Updated this PR to instead add support for 1.21.3

File: build.gradle [hunk 2]
```diff
@@ -30,7 +30,7 @@ dependencies {
 	shadow group: 'org.bstats', name: 'bstats-bukkit', version: '3.0.2'
 	shadow group: 'net.kyori', name: 'adventure-text-serializer-bungeecord', version: '4.3.2'
 
-	implementation group: 'io.papermc.paper', name: 'paper-api', version: '1.21-R0.1-SNAPSHOT'
+	implementation group: 'io.papermc.paper', name: 'paper-api', version: '1.21.1-R0.1-SNAPSHOT'
```
Comment (sovdeeth) [inline build.gradle:33]: last i checked, 1 is not equal to 3

**Review by sovdeeth — CHANGES_REQUESTED** [review]

File: build.gradle (hunk 2 above)
Comment (EquipableMC) [reply build.gradle:33]: Like I said, I haven't pushed changes yet. There is no dev server bundle available yet. Once that is I will push the changes.

Comment (sovdeeth) [reply build.gradle:33]: Oh please don't update the pr saying it does something it doesn't yet do, then

Comment (EquipableMC) [reply build.gradle:33]: Sorry, will keep note of that in the future.

**Comment by EquipableMC** [comment]
Closing this PR as tests are being added in the change in #7173.

**Pull request closed without merging**